Makefile.d/docker.mk

```
@@ -149,6 +149,15 @@ docker/build/operator/helm:
 	    -t $(REPO)/$(HELM_OPERATOR_IMAGE) . \
 	    --build-arg OPERATOR_SDK_VERSION=$(OPERATOR_SDK_VERSION)
 
+.PHONY: docker/name/loadtest
+docker/name/loadtest:
+	@echo "$(REPO)/$(LOADTEST_IMAGE)"
+
+.PHONY: docker/build/loadtest
+## build loadtest image
+docker/build/loadtest:
```

It should depend on base image.
```suggestion
docker/build/loadtest: docker/build/base
```